db/seed/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM mongo
+
+COPY data.json ./data.json
+CMD mongoimport --host db --port 27017 \
+    --db teams --collection team \
```

Comment:
The '--collection team' part indicates the data should be imported from a collection called 'team', but I'm not sure where this is. All I could see was the 'db/seed/data.json' file, but I couldn't see the word 'team' there (or not in a way mongoimport will identify, anyway). Am I missing something?

Reply:
You are, a little - `--collection team` is the target, not the source. The entire file will be loaded into a new collection (think 'table') called `team`.

Which is, inarguably, the wrong name for the collection (it should be 'teams'), but I'm happy to patch those in later to be honest